cover/v1/cover.proto

```diff
@@ -2295,6 +2295,8 @@ message CostItem {
   string unit = 12;
   // Category. Contains the category to which a service belongs. Only used if groupBy is set to category.
   string category = 13;
+  // Tagged. Tagged for those with tag, otherwise Untagged.
+  string tagged = 14;
```

Comment:
I am thinking, what if we use boolean true if tagged otherwise untagged?

Reply:
Okay, I'll change this one @tituscarl